frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -202,9 +257,9 @@ export class ClusterOverview extends React.Component {
           },
           inventoryData: getInventoryData(resources), // k8s object loaded via WithResources
           healthData: this.state.healthData,
+          capacityStats: this.state.capacityStats,
```

Comment:
now, when Capacity card supports external loading component, we should use tectonic's LoadingInLine

Reply:
done